test/fixtures/insecure-registries/python-local-deployment.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: python-local
+  namespace: services
+  labels:
+    app.kubernetes.io/name: python-local
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: python-local
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: python-local
+    spec:
+      containers:
+      - image: kind-registry:5000/python:rc-buster
+        imagePullPolicy: Always
+        name: nginx-local
```

Comment:
This container name is a bit misleading but not a big deal 😄

Reply:
oops